shutter/low-level.md

```diff
@@ -261,19 +261,24 @@ Transactions `txs[j]` for `j >= 1` are a subset of `decrypted_transactions`. The
 
 ### Encrypting RPC Server
 
-> :construction: :construction: :construction:
->
-> Todo
->
-> :construction: :construction: :construction:
+The Encrypting RPC Server is an HTTP server that exposes a modified version of the Ethereum JSON RPC protocol as defined [here](https://ethereum.github.io/execution-apis/api-documentation/). It implements all methods from the `eth` namespace as specified, except for `eth_sendRawTransaction`. It may optionally implement any other method.
 
-### Decryption Key Relay
+The server is configurable with the following values:
 
-> :construction: :construction: :construction:
->
-> Todo
->
-> :construction: :construction: :construction:
+- an ECDSA private key `private_key` corresponding to address `address`
+- a non-negative integer `keyper_set_change_lookahead`
+
+#### eth_sendRawTransaction
+
+The server exposes a method `eth_sendRawTransaction` that behaves differently to the standard. It takes a hex encoded, `0x` prefixed string `h` as its sole parameter. If not exactly one parameter is provided, the server responds with an error with code `-32602`. If `h` is not the hex encoding of a byte string `b` or `b` is not the RLP encoding of a valid transaction `tx`, it returns an error with code `-32602`.
```

Comment:
can you use regular names instead of single letters `h`, `b`, `s`? Should make it a little more readable.